src/index.js

```diff
@@ -0,0 +1,7 @@
+import { SuiTealiumTracker } from './SuiTealiumTracker'
+/**
+ * The install.js file will have the ownership of attach listeners to the dom elements that have the tealiumTag
+ * data tag.
+ * @param {Object} params
+ */
+module.exports = params => new SuiTealiumTracker(params).init()
```

Comment:
Better export it named.

Reply:
I am totally agree to change it but...id like to know why 🤓 , is for code styling, performance?

Reply:
Well, in order to avoid arbitrary names when importing it. Only code styling.

Mainly code styling, as it let you the follow line.
`import Whatever from '@s-ui/tealium-tracker'`